Multisig/Assets.xcassets/Key Type/Icon/ico-key-type-keystone.imageset/Contents.json

```diff
@@ -1,17 +1,7 @@
 {
   "images" : [
     {
-      "filename" : "ico-add-keystone.pdf",
-      "idiom" : "universal"
-    },
-    {
-      "appearances" : [
-        {
-          "appearance" : "luminosity",
```

Comment:
@zhiying-fan I think it will still better if we have dark mode version of the logo

Reply:
| Light | Dark |
| - | - |
| ![Simulator Screenshot - iPhone 14 - 2023-08-24 at 20 13 36](https://github.com/safe-global/safe-ios/assets/38421753/1dd149b1-a460-46b9-a20e-b3811fd020e8) | ![Simulator Screenshot - iPhone 14 - 2023-08-24 at 20 13 50](https://github.com/safe-global/safe-ios/assets/38421753/5cf451b8-e7fa-48a9-b2f5-070da36ffe98) |

It looks good for both light and dark. Could you confirm? Thanks

Reply:
Actually comparing to other rows, I don't think so, for all other key types we have two different icons for light/dark modes which comply to our styling colors, and I would still do the same here

Reply:
Thanks for raising your concern, I will let our designer review the UI again. Will come back later.

Reply:
@MouazAlzahabi I added dark logo and update the screenshots above, could you please have a look again? It is better than before.